cmd/uhc/cluster/login/login.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package login
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/openshift-online/uhc-cli/pkg/config"
+	"github.com/openshift-online/uhc-cli/pkg/util"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	debug bool
+	user  string
+	pass  string
+}
+
+var Cmd = &cobra.Command{
+	Use:   "login CLUSTERID",
+	Short: "login to a cluster",
+	Long:  "login to a cluster by CLUSTERID using openshift oc",
+	Run:   run,
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.BoolVar(&args.debug, "debug", false, "Enable debug mode.")
```

Comment:
Pleae put each argument to these methods in a separate line, as we do in all the other commands:

```go
flags.BoolBar(
    &args.debug,
    "debug",
    false,
    "Enable debug mode.",
)
```

Reply:
done